### ISSUE TITLE
 Reconstitution China History Plan

### ISSUE THREAD

**Issue by yhpdoit** [Wolferos/Hearts-of-Iron-IV-The-Great-War#155]
Second Guangzhou Uprising - April,13,1911 - Simple Event

Railway Protection Movement - August,11,1911 - Simple Event

Wuchang Uprising - 10,October,1911

User Choice :

choose support Qing/Republic of China

Result :

Start Xinhai Revolution

Start Civil war(Qing dynasty - Government of the Republic of China)

Release Contry = Tannu Tuba,Great Mongolian State,Tibet\

Return of Yuan Shikai - October,1,1911

Condition :

Civil war in Qing

User Choice :

Co-opertaion With Yuan Shikai / Never

Result :

If user choose Co-opertaion With Yuan Shikai
Remove general = Yuan Shikai from Qing
Republic of China is rulled by Yuan Shikai

China North–South Conference - December,18,1911

Condition :

Co-opertaion With Yuan Shikai
Yuan Shikai is ruler of Republic of China

User Choice(if play Qing) :

Accept / Refuse

Result:

if Qing Accept :

Qing Surrander - End Xinhai Revolution
Roc annex all Qing land

if Qing Rufuese:

heavy rebel upriseing.

Hongxian Emperor - December,12,1915

Condition :

China Ruler = Yuan Shikai

Result :

China New Name : Hongxian Emperor

China New Rulling Party = Yuan Shikai's Party(Authoritarian)

Death of Yuan Shikai - 6,June,1916

Condition :

China Ruler = Yuan Shikai
or
Death of Sun Yat-sen Event

User choice :

user can select support controy

Result :

China Warloard Age Start.
China Serparted
Anhui clique 

Ruler Duan Qirui

Control ANHUI,HUBEI,HUNAN,SHAAANXI

Zhili clique

Ruler Wu Peifu

Capital Beijng

Control BEJING EAST HUBEI,HUBEI,SHANGDONG,HENAN,JIANGSU

Fengtian clique 

Ruler Zhang Zuolin

Capital Hsinking

Control JEHOL,CHAHAR.LIAONING,LIAOTUNG,KIRIN,SUNGKIANG,

HEILUNGKIANG,SOUTH CHAHAR,CHARHAR

Shanxi clique 

Ruler Yan Xishan

Capital Taiyuan

Control SHANXI,SUIYUAN

Yunnan clique

Ruler Long Yun

Capital Kuming

Control YUNNAN

Guangxi Clique

Ruler Li Zongren

Capital Nanning

Control GUANGXI,GUANGDONG,NANNING,GUANGZHOU,HAINAN

Sianking

Ruler Sheng Shical

Capital Urumqi

Control TAKLAMAKAKAN,YARKAND,URUMQI,DZUNGARIA

Xibel san ma

Ruler Ma Bufang

Capital Golmud

Control Gansu,QingHai,Ningxia

Sichuan clique

Ruler Liu Xiang

Capital Chongqing

Control XIKANG,SICHUAN,GUIZHOU

Kuomintang

Ruler Sun Yat-sen(if Sun Yat-sen's Death, Control Chiang Kai-shek)
Captail Nanjing
SHANGHAI,ZHEJIANG,FUJIAN,JIANGXI

May Fourth Movement - May,4,1919

Condition : 

Treaty of Versailles Exist

Result :

Contry in Chain Gets Daily Communist Support 0.10 Until 1925

First United Front

Condition : 

May Fourth Movement

Result :

Kuomintang Add Party - Communist (leader Mao Zedong)

Kuomintang Gains National Sprit First United Front 

(Division Recovery Rate 10%.Factory Output 10%,Daily Communist Support 0.5)

Death of Sun Yat-sen's Death - 1925.3.12

Kuomintang Ruler Chiang Kai-shek

Northern Expedition

Annex Fengtian clique
Annex Sichuan clique

Shanghai massacre (Kill the Communist Party)

Remove National Sprit First United Front
Add National Sprit Comuninist Upring

The Long March Start - Nov,15,1931
The Long March End

Comuinist Chnia Creation
Condition :

People's Republlic of China (Ruler Mao Zedong,Captail Yan'an)

Mukden Incident September - Sep,18, 1931

Make Manchukuo
Make Mengjiang

**Comment by DevilIris** [comment]
Can you code it ?

**Comment by yhpdoit** [comment]
one day or other. no time limit for this :)

**Comment by DevilIris** [comment]
But can you code the events?

**Comment by yhpdoit** [comment]
What means?

**Comment by MasterWaifuman** [comment]
that means can you write the code for them like that?
![examplexinhai](https://user-images.githubusercontent.com/30695036/32655995-babf723c-c618-11e7-8888-b20e26d52df9.PNG)

**Comment by MasterWaifuman** [comment]
or you just give plan what to do?

**Comment by yhpdoit** [comment]
i can. my problem is git and english
they are diffcult,not hoi4 moding.

**Comment by yhpdoit** [comment]
First at all, i suggest something 

\event\WW1_Xinhai.txt line 188
set_national_unity = 0.1 

this line must be removed.  10% NU is QIN surrender soon. so, nobody cannot show title=ww1_xinhai.4.(Return of Yuan Shikai). war is end without event. and no event progress.

**Comment by yhpdoit** [comment]
I'd like to do it that myself, but git is so difficult, :)

**Comment by MasterWaifuman** [comment]
I don't think it is so difficult. You can do things straight through the original repository or like me and other guys do it through DevilIris repository.

**Comment by MasterWaifuman** [comment]
Just train with your own repository

**Comment by MasterWaifuman** [comment]
> this line must be removed. 10% NU is QIN surrender soon. so, nobody cannot show title=ww1_xinhai.4.(Return of Yuan Shikai). war is end without event. and no event progress.

I think all lines should be removed and reworked completely. All civil wars should be reworked. This is one of our main goals.

**Comment by yhpdoit** [comment]
No,no, i cannot train with my own repository. i need how to contribute other's repository.

https://imgur.com/a/85ri0

show this image. i click publish branch, and i received you have no permission.
my own repository is no problem. how i sovle it?

**Comment by MasterWaifuman** [comment]
I don't use desktop github version. I just edit files in notepad++, test it, copy all things and paste in files in the repository.

**Comment by MasterWaifuman** [comment]
But i think it it id similar for desktop too

**Comment by MasterWaifuman** [comment]
in browser version you have that thing if you work through original repo. That means you need to create pull request.

**Comment by MasterWaifuman** [comment]
But i work through DevilIris repo. So i have less such problems

**Comment by DevilIris** [comment]
we need to hurry in the main project the next update is in 2 weeks most
likely

2017-11-10 13:34 GMT-06:00 Master Waifuman <notifications@github.com>:
